hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/BucketArgs.java

```diff
@@ -61,6 +62,11 @@
   private long quotaInBytes;
   private long quotaInNamespace;
 
+  /**
+   * Bucket Type.
+   */
+  private BucketType bucketType;
```

Comment:
we should have a default value for the bucket type, or is it a required arg ?

Reply:
Yes, I'll add a default value to bucketType.